### ISSUE TITLE
NVDA seems not to support Notepad++ 8.3

### ISSUE THREAD

**Issue by aaclause** [nvaccess/nvda#13311]
### Steps to reproduce:
1. Start Notepad++ 8.3 (downloadable [here](https://notepad-plus-plus.org/downloads/v8.3/)), you should have a blank document.
2. Write some text (just one character is enough).

Alternatively, with an installed version you can just open a file from explorer.

### Actual behavior:
Notepad++ crashes (the app closes), NVDA fails to retrieve file content.

### Expected behavior:
No crash, access to file content.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-24583,61000de2

#### Windows version:
10 21H2

#### Name and version of other software in use when reproducing the issue:
Notepad++ 8.3 (installed and portable versions)

#### Other information about your system:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. But no problem with Notepad++ 8.2.1 and earlier.

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by cary-rowen** [comment]
I can also reproduce this.
@seanbudd 
@feerrenrut 
can you take a look at this?

**Comment by aaclause** [comment]
Out of curiosity, I've just tested with JAWS 2022: also no access to file contents but Notepad++ doesn't crash.

**Comment by MichelSuch** [comment]
Also does not crash with Narrator, but cannot read file content.

With NVDA, it crashes as soon as you load a fiel or when typing text in 
an empty file.




Michel Such ***@***.***>
------------------------------------------------------------------------
Expert NVDA certifié 2019
Le 08/02/2022 à 07:27, André-Abush Clause a écrit :

**Comment by aaclause** [comment]
@MichelSuch yes but Narrotor doesn't work with Notepad++ 8.2 also.

**Comment by cary-rowen** [comment]
Does this mean we are going to write an issue in the notepad++ repository

**Comment by TheQuinbox** [comment]
I opened an issue before I knew this was related to NVDA actually, https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11133

**Comment by rbakhshi72** [comment]
Today, I downloaded Notepad++ 8.3, and I understood that it is not accessible for us. It means that when I type some Characters and navigate to read what every I typed, but NVDA doesn’t read anything. I don’t have this issue in 8.2, too.

**Comment by lukaszgo1** [comment]
If someone is interested in testing you can do so with [this try build](https://ci.appveyor.com/api/buildjobs/vu4s4chopddi8gqj/artifacts/output%2Fnvda_snapshot_pr13364-24749%2C17af7985.exe)
